lint-staged.config.js

```diff
@@ -0,0 +1,5 @@
+export default {
+  "*.js": ["npm run lint:js --", "npm run format --"],
```

Comment:
Seems like by adding `npm run format` to both globs now, would it would now be running twice, at least for JS files?

Is it possible to keep `format` applied only to the wildcard glob?